scheduler/feasible.go

```diff
@@ -551,7 +552,11 @@ func checkConstraint(ctx Context, operand string, lVal, rVal interface{}, lFound
 	case structs.ConstraintAttributeIsNotSet:
 		return !lFound
 	case structs.ConstraintVersion:
-		return lFound && rFound && checkVersionMatch(ctx, lVal, rVal)
+		cfac := newVersionConstraintParser(ctx)
```

Comment:
What is `cfac`? Can't think of what that is an acronym for. Maybe `p` for parser, that is what is declared in the receiving function as well?

Reply:
Oops! Used to be called ConstraintFactory. Fixed cfac.